README.md

```diff
@@ -32,14 +32,15 @@ More information on terminology can be found [here](docs/terminology.md).
 To populate that database
 ```
 ## Initialize DBs
-./phg initdb --dbpath /path/to/dbs
+./phg initdb --db-path /path/to/dbs
 
 ## Build VCF data
 ./phg create-ranges --gff my.gff --boundary gene --pad 500 -o /path/to/bed/file.bed
-./phg align-assemblies --gff anchors.gff --ref Ref.fa -a assembliesList.txt -o /path/to/temp
+./phg align-assemblies --gff anchors.gff --reference-file Ref.fa -a assembliesList.txt -o /path/to/temp
+./phg agc-compress --db-path /path/to/dbs --reference-file /my/ref.fasta --fasta-list /my/assemblyFastaList.txt 
 ./anchorwave (ref.fasta, asm.fasta) -o /path/to/maf/files/
```

Comment:
Do we want to keep this direct anchorwave call?

Reply:
No I will remove from the doc